templates/hotgraphicPopupToolbar.jsx

```diff
@@ -0,0 +1,82 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function HotgraphicPopupToolbar(props) {
+  const a11yConfig = Adapt.config.get('_accessibility');
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const {
+    onCloseClick,
+    onControlClick,
+    shouldEnableBack,
+    shouldEnableNext,
+    itemCount,
+    backLabel,
+    nextLabel,
+    _hidePagination
+  } = props;
+
+  return (
+    <div className={classes([
+      'hotgraphic-popup__toolbar',
+      _hidePagination && 'hide-pagination'
+    ])}>
+
+      {!_hidePagination &&
+      <div className={classes([
+        'hotgraphic-popup__nav',
+        !shouldEnableBack && 'first',
+        !shouldEnableNext && 'last'
+      ])}
+      >
+
+        <button
+          className={classes([
+            'btn-icon hotgraphic-popup__controls back',
+            !shouldEnableBack && 'is-disabled'
+          ])}
+          aria-label={backLabel}
+          aria-disabled={!shouldEnableBack || null}
+          onClick={onControlClick}
+          data-direction='back'
+        >
+          <span className="icon" aria-hidden="true" />
+        </button>
+
+        <div
+          className="hotgraphic-popup__count"
+          dangerouslySetInnerHTML={{ __html: itemCount }}
```

Comment:
Needs aria-hidden set to true as the new carousel behavior from wcag changes read out the numbering on the back and next button.

Reply:
Done.